src/cdk/a11y/aria-describer/aria-describer.ts

```diff
@@ -186,6 +186,7 @@ export class AriaDescriber implements OnDestroy {
       messagesContainer = this._document.createElement('div');
       messagesContainer.id = MESSAGES_CONTAINER_ID;
       messagesContainer.classList.add('cdk-visually-hidden');
+      messagesContainer.style.visibility = 'hidden';
```

Comment:
Can you add a comment here like

```
// We add `visibility: hidden` in order to prevent text in this container
// from being searchable by the browser's Ctrl + F functionality. 
// Screen-readers will still read the description for elements with
// aria-describedby even when the description element is not visible.
```